File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/event/exception/AlreadyCalculatingStatusException.java
package band.gosrock.domain.domains.event.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class AlreadyCalculatingStatusException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new AlreadyCalculatingStatusException();

    private AlreadyCalculatingStatusException() {
        super(EventErrorCode.ALREADY_CALCULATING_STATUS);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/event/exception/AlreadyCalculatingStatusException.kt
package band.gosrock.domain.domains.event.exception

import band.gosrock.common.exception.DuDoongCodeException

class AlreadyCalculatingStatusException private constructor() : DuDoongCodeException(EventErrorCode.ALREADY_CALCULATING_STATUS) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = AlreadyCalculatingStatusException()
    }
}
